### PR TITLE
[cleanup] Format improvements

### PR DESCRIPTION
**Pull request by ghost**
Solves ipa320/cob_navigation#69

Actual changes are in the commit before the last.

### CODE REVIEW

File: cob_map_accessibility_analysis/ros/include/cob_map_accessibility_analysis/map_accessibility_analysis_server.h [hunk 1]
```diff
@@ -1,62 +1,19 @@
...
-* notice, this list of conditions and the following disclaimer. \n
-* - Redistributions in binary form must reproduce the above copyright
-* notice, this list of conditions and the following disclaimer in the
-* documentation and/or other materials provided with the distribution. \n
-* - Neither the name of the Fraunhofer Institute for Manufacturing
-* Engineering and Automation (IPA) nor the names of its
-* contributors may be used to endorse or promote products derived from
-* this software without specific prior written permission. \n
+/*
+* Copyright (c) 2017 Fraunhofer Institute for Manufacturing Engineering and Automation (IPA)
```
Comment (mgruhler) [inline cob_map_accessibility_analysis/ros/include/cob_map_accessibility_analysis/map_accessibility_analysis_server.h]: please make all copyrights 2016-2017

File: cob_map_accessibility_analysis/ros/src/map_accessibility_analysis_client.cpp [hunk 2]
```diff
@@ -1,206 +1,152 @@
...
+  serviceAvailable &= ros::service::waitForService(perimeter_service_name, 5000);
+  serviceAvailable &= ros::service::waitForService(polygon_service_name, 5000);
+
+  // only proceed if the service is available
+  if (serviceAvailable == false)
+  {
+    ROS_ERROR("The services could not be found.\n");
+    return -1;
+  }
+  ROS_INFO("The service servers are advertised.\n");
```
Comment (mgruhler) [inline cob_map_accessibility_analysis/ros/src/map_accessibility_analysis_client.cpp]: You don't need a  `\n` in ROS logging. Please remove.

**Review by mgruhler — CHANGES_REQUESTED** [review]
only minor issues...

@ipa-rmb @ipa-bnm:

@mig-em tackled #69.
If you are working on this package, please let us know. Otherwise we'll end up with merge conflicts.

**Comment by ipa-rmb** [comment]
I am not working on this package at the moment, you can go ahead.

**Review by mgruhler — APPROVED** [review]

**Comment by mgruhler** [comment]
someone else wants to review? @ipa-fxm? @ipa320/navigation-push?

**Review by fmessmer — APPROVED** [review]

**Pull request merged**